live-build/misc/migration-scripts/dx_execute

```diff
@@ -131,9 +131,17 @@ echo "Running MDS Command: '$command'"
 /opt/delphix/server/bin/mds_client -- -c "$command" ||
 	die "Failed mds_client command."
 
+#
+# Let the app-stack know that an "OS migration" upgrade is being performed.
+#
+cat <<-EOF >"/var/dlpx-update/upgrade.properties" || die "failed to create upgrade.properties"
```

Comment:
I'm not sure what is the correct structure here as I've seen other places where we do the following:

```
cat <-EOF >"/var/dlpx-update/upgrade.properties" ||
    UPGRADE_TYPE=OS_MIGRATION
EOF 
    die "failed to create upgrade.properties"
```

We should standardize on a particular convention

Reply:
I agree. I have seen many different ways of that being done. Unfortunately this is not being enforced in `shfmt` so we'd need to somehow add our own rules.

Reply:
Often, the "die" command is on a new line so it fits within the 80 column limit. If we don't need another line to fit in that limit, then I don't see any reason to use another line.

Reply:
I've put the die on a different line to make it fit within the 80 characters limit.